LSP.sublime-settings

```diff
@@ -537,6 +537,16 @@
       //   "diagnostics": true
       // },
     },
+    "sorbet": {
+      "command": ["srb", "tc", "--typed", "true", "--enable-all-experimental-lsp-features", "--lsp", "--disable-watchman"],
+      "languageId": "ruby",
+      "scopes": ["source.ruby"],
```

Comment:
Could you also keep this in sync with solargraph's settngs? Solargraph includes `source.ruby.rails` here. After that, Looks mergeable to me.

Reply:
is done